sylt-parser/src/statement.rs

```diff
@@ -113,6 +116,26 @@ pub struct Statement {
     pub kind: StatementKind,
 }
 
+pub fn path<'t>(ctx: Context<'t>) -> ParseResult<'t, String> {
+    let mut ctx = ctx;
+    let mut result = String::new();
+    expect!(ctx, T::Slash | T::Identifier(_), "Expected file path");
```

Comment:
```suggestion
    expect!(ctx, T::Slash | T::Identifier(_), "Expected identifier or slash at start of use path");
```
Maybe something like this? I'm also not sure I like the word `file path` - but that's easily changed later.